ansible/roles/sidekiq/templates/config/systemd/user/sidekiq@.service

```diff
@@ -8,7 +8,8 @@ PartOf=sidekiq.service
 [Service]
 Type=simple
 Environment=RAILS_ENV=production
-Environment=MALLOC_ARENA_MAX=2
+Environment=LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so
+Environment=MALLOC_CONF="narenas:2,background_thread:true,thp:never,dirty_decay_ms:1000,muzzy_decay_ms:0"
```

Comment:
This config is borrowed from [here](https://twitter.com/nateberkopec/status/1442894624935137288) and [here](https://github.com/code-dot-org/code-dot-org/blob/staging/cookbooks/cdo-jemalloc/attributes/default.rb). jemalloc 5 (current version) does not work well with ruby 2.7 without this (it performs more or less the same as malloc).